### ISSUE TITLE
Issues regarding the segmentation performance of ultrasound images.

### ISSUE THREAD

**Issue by ZhichengQian1** [TimesXY/Cold-SegDiffusion#4]
When I train the model, my training loss is very low, but the performance on both the training set and the validation set is very poor.
![image](https://github.com/user-attachments/assets/77a83c4a-3041-4e0a-9892-baf5c45638f3)
Is this due to the noise t being too short? Or is it caused by ultrasound image artifacts, or is it because the number of training epochs is insufficient?

**Comment by ZhichengQian1** [comment]
![image](https://github.com/user-attachments/assets/575c52f1-bd5e-44bd-bc93-5b1ead01e3f3)
This refers to the segmentation performance on the validation set.

**Comment by TimesXY** [comment]
Please consider reducing the image size to 128 and make sure that the number of training epochs is sufficient.
